api/src/main/java/io/hyperfoil/function/SerializableToIntFunction.java

```diff
@@ -1,7 +1,6 @@
 package io.hyperfoil.function;
 
-import java.io.Serializable;
 import java.util.function.ToIntFunction;
 
-public interface SerializableToIntFunction<T> extends ToIntFunction<T>, Serializable {
+public interface SerializableToIntFunction<T> extends ToIntFunction<T>, SerializableToObjectFunction<T> {
```

Comment:
Classes in `i.h.function` are meant to be a straight extension of `java.util.function.*` with added `Serializable`. Besides, extending `SerializableToObjectFunction` here makes little sense.

Reply:
Removed `SerializableToObjectFunction `